### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Marchenkov**
- [DEMO LINK](https://Marchenkov.github.io/layout_stars/)
- [TEST REPORT LINK](https://Marchenkov.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,55 @@
...
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (anastasiiavorobiova) [inline src/style.css:55]:
```suggestion

```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```
Comment (anastasiiavorobiova) [inline src/style.css:37]:
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
... ,
.stars--5 .stars__star:nth-child(-n + 5) {
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done! Consider grouping selectors with commas

**Comment by Marchenkov** [comment]
Done changes

**Pull request closed without merging**